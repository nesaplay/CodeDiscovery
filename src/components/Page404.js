import React from 'react'
import { Link } from 'react-router-dom'
import { Container } from 'semantic-ui-react'

const Page404 = () => {
	return (
		<Container>
			<div>
				<h1>Psst! You shouldn't be here!</h1>
				<p>
					Go back <Link to="/">HOME</Link> and search for more
					repositories.
				</p>
			</div>
		</Container>
	)
}

export default Page404
