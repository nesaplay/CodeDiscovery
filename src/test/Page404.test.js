import React from 'react'
import Page404 from '../components/Page404'

it('renders a component', () => {
	const wrapper = shallow(
        <Page404 />
    )

	expect(wrapper).toMatchSnapshot()
})
