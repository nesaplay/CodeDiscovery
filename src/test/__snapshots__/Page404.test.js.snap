// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a component 1`] = `
<Container>
  <div>
    <h1>
      Psst! You shouldn't be here!
    </h1>
    <p>
      Go back 
      <Link
        replace={false}
        to="/"
      >
        HOME
      </Link>
       and search for more repositories.
    </p>
  </div>
</Container>
`;
